jasmine.js

```diff
@@ -0,0 +1,39 @@
+describe("array numbers", function() {
+
+	it("check elements from array", function() {
+	 	expect(tests.firstTest()).toEqual(jasmine.any(Number));
+  	});
+
+});
+
+describe("firstTest", function() {
+
+	it("check for even numbers", function() {
+    		expect(tests.numbers).toEqual(jasmine.arrayContaining([4, 6]));
```

Comment:
This is a weird assertion, you are asserting that your initial array contains 4 and 6, what is the purpose of this?